### ISSUE TITLE
Submission: Stochastic (Combined Heat and Power) Anaerobic Digestion Economic Assessment Tool (SADEAT)

### ISSUE THREAD

**Issue by whedon** [openjournals/joss-reviews#65]
**Submitting author:** @AbdulSalam2016 (<a href="http://orcid.org/0000-0002-5784-4576">YAKUBU ABDUL-SALAM</a>)
**Repository:** <a href="https://github.com/AbdulSalam2016/SADEAT-Stochastic-Combined-Heat-and-Power-Anaerobic-Digestion-Economic-Assessment-Tool/releases/latest" target ="_blank">https://github.com/AbdulSalam2016/SADEAT-Stochastic-Combined-Heat-and-Power-Anaerobic-Digestion-Economic-Assessment-Tool/releases/latest</a>
**Version:** v1.0.0

## Status

[![status](http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943/status.svg)](http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943)

Status badge code:

```
HTML: <a href="http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943"><img src="http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943/status.svg"></a>
Markdown: [![status](http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943/status.svg)](http://joss.theoj.org/papers/1a701d658ed4d114267f1f6668ae3943)
```
**Reviewers and authors**:

Please avoid lengthy details of difficulties in the review thread. Instead, please create a new issue in the <a href="https://github.com/AbdulSalam2016/SADEAT-Stochastic-Combined-Heat-and-Power-Anaerobic-Digestion-Economic-Assessment-Tool/releases/latest" target="_blank">target repository</a> and link to those issues (especially acceptance-blockers) in the review thread below. (For completists: if the target issue tracker is also on GitHub, linking the review thread in the issue or vice versa will create corresponding breadcrumb trails in the link target.)

### Conflict of interest

- [ ] As the reviewer I confirm that there are no conflicts of interest for me to review this work (such as being a major contributor to the software).

### General checks

- [ ] **Repository:** Is the source code for this software available at the <a target="_blank" href="https://github.com/AbdulSalam2016/SADEAT-Stochastic-Combined-Heat-and-Power-Anaerobic-Digestion-Economic-Assessment-Tool/releases/latest">repository url</a>?
- [ ] **License:** Does the repository contain a plain-text LICENSE file with the contents of an [OSI approved](https://opensource.org/licenses/alphabetical) software license?
- [ ] **Version:** Does the release version given match the GitHub release (v1.0.0)?

### Functionality

- [ ] **Installation:** Does installation proceed as outlined in the documentation?
- [ ] **Functionality:** Have the functional claims of the software been confirmed?
- [ ] **Performance:** Have any performance claims of the software been confirmed?

### Documentation

- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **Installation instructions:** Is there a clearly-stated list of dependencies? Ideally these should be handled with an automated package management solution.
- [ ] **Example usage:** Do the authors include examples of how to use the software (ideally to solve real-world analysis problems).
- [ ] **Functionality documentation:** Is the core functionality of the software documented to a satisfactory level (e.g. API method documentation)?
- [ ] **Automated tests:** Are there automated tests or manual steps described so that the function of the software can be verified?
- [ ] **Community guidelines:** Are there clear guidelines for third parties wishing to 1) Contribute to the software 2) Report issues or problems with the software 3) Seek support

### Software paper

- [ ] **Authors:** Does the `paper.md` file include a list of authors with their affiliations?
- [ ] **A statement of need:** Do the authors clearly state what problems the software is designed to solve and who the target audience is?
- [ ] **References:** Do all archival references that should have a DOI list one (e.g. papers, datasets, software)?

**Comment by arfon** [comment]
👋 @AbdulSalam2016, thanks for your submission to JOSS.

Before we can proceed with the review there are a couple of things that need fixing up with your submission.

- Please reformat your paper (`paper.md`) to be formatted _exactly_ as shown here https://raw.githubusercontent.com/arfon/fidgit/master/paper/paper.md. In particular, the paper metadata needs to be formatted properly before we can proceed.
- I see that you've attached both the source code and the compiled executable to the release on GitHub: https://github.com/AbdulSalam2016/SADEAT-Stochastic-Combined-Heat-and-Power-Anaerobic-Digestion-Economic-Assessment-Tool/releases/latest . Please make sure that the source code is available in a browsable interface (e.g. a GitHub repository) so that the reviewers can view the code. Attaching a zipped archive of the code to a release is going to make it very hard for our reviewers to process this submission.

Please reply on this thread when you've made these updates and we can proceed with the review 😄

**Comment by AbdulSalam2016** [comment]
Thank you very much for considering my submission to your journal, Arfon. I have to clarify however that I do not include the source code in my submission, only the executable. I seek to maintain control of the source code although the current and future version of the software will be free. Does non-inclusion of the source code have implications for submission to your journal? Thank you very much.

**Comment by arfon** [comment]
> Does non-inclusion of the source code have implications for submission to your journal? Thank you very much.

Hi @AbdulSalam2016. Thanks for clarifying. Unfortunately this doesn't meet the requirements of the journal as the source code is not openly available for review. I'll close this review issue and remove the submission from the JOSS site.

If you every change your mind we'd welcome a re-submission.

**Comment by AbdulSalam2016** [comment]
Hi @arfon. Thank you very much. Will consider this.
